docker/Dockerfile

```diff
@@ -97,7 +97,7 @@ COPY deployments/gpu-operator/charts/node-feature-discovery/crds/nfd-api-crds.ya
 # Install / upgrade packages here that are required to resolve CVEs
 ARG CVE_UPDATES
 RUN if [ -n "${CVE_UPDATES}" ]; then \
-        yum update -y ${CVE_UPDATES} && \
+        dnf update -y ${CVE_UPDATES} && \
```

Comment:
Question -- just curious, does `yum` not exist in the ubi9 images?

Reply:
It does, I made this change because I thought we should be consistent with our yum/dnf usage in the Dockerfile. Moreover, `dnf` is the recommended pkg manager tool over `yum`